redis/client_test.go

```diff
@@ -67,6 +68,11 @@ func TestParse(t *T) {
 	assert.Equal(t, ErrorReply, r.Type)
 	assert.Equal(t, "ERR unknown command 'foobar'", r.Err.Error())
 
+	// wrongtype reply
+	r = parseString("-WRONGTYPE Operation against a key holding the wrong kind of value\r\n")
+	assert.Equal(t, ErrorReply, r.Type)
+	assert.Equal(t, "WRONGTYPE Operation against a key holding the wrong kind of value", r.Err.Error())
+
```

Comment:
And since the definition of WRONGTYPE seems like a bad request, we should not have any bad command code. I just want to make sure the driver can still parse it.